recipe/meta.yaml

```
@@ -34,24 +38,31 @@ requirements:
     - patch     # [not win]
     - m2-patch  # [win]
   host:
-    - {{n}}expat
+    - expat 2.5.0
     - {{n}}gcc-libs          # [win]
   run:
     - {{n}}gcc-libs          # [win]
-    - {{n}}expat
+    - expat >=2.5.0,<3.0a0
```

```suggestion
    - expat  # pin through run_exports
```
`expat` already has a run_exports: https://conda-metadata-app.streamlit.app/?q=pkgs%2Fmain%2Flinux-64%2Fexpat-2.5.0-h6a678d5_0.conda